exopy_hqc_legacy/instruments/drivers/visa/yokogawa.py

```diff
@@ -142,7 +142,7 @@ def current(self):
         NB: does not check the current function.
 
         """
-        current = self.query(":SOURce:LEVel?")
+        current = 1e6*self.query(":SOURce:LEVel?")
```

Comment:
Everything else tend to be in bare SI unit why did you choose to introduce something different here ?

Reply:
It seems that the Yoko speaks uA instead of A so we do the conversion in the driver.

Reply:
Weird. I am going to trust the people with the instrument

Reply:
SInce you were unsure, I checked myself and I can confirm the fact that the Yoko indeed expects and returns uA for all current related functions!